app/src/components/course-form/courseFormDirective.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
-angular.module('appCourseDirective')
+angular.module(
+    'appCourseFormDirective',
+    ['appCourseFormController']
+)
     .directive('appCourseForm', function() {
         return {
             scope: false,
             templateUrl: '/src/components/course-form/courseForm.tpl.html',
+            controller: [
+                '$scope', 'coursesService',
```

Comment:
Could you use row or column style for array

Reply:
OK. I fixed it.